src/components/dialogs/network-modifications/line-attach-to-voltage-level/line-attach-to-voltage-level-form.js

```diff
@@ -122,19 +100,28 @@ const LineAttachToVoltageLevelForm = ({
         <TextInput name={LINE2_NAME} label={'Line2Name'} />
     );
 
+    const busbarSectionOptions = useMemo(() => {
+        if (voltageLevelIdWatch === voltageLevelToEdit?.equipmentId) {
+            return voltageLevelToEdit.busbarSections;
+        }
+    }, [voltageLevelIdWatch, voltageLevelToEdit]);
+
     const connectivityForm = (
         <ConnectivityForm
             label={'AttachedVoltageLevelId'}
             withPosition={false}
             withDirectionsInfos={false}
             voltageLevelOptions={allVoltageLevelOptions}
-            newBusOrBusbarSectionOptions={voltageLevelToEdit?.busbarSections}
+            newBusOrBusbarSectionOptions={busbarSectionOptions}
             studyUuid={studyUuid}
             currentNode={currentNode}
-            onVoltageLevelChangeCallback={onVoltageLevelChange}
         />
     );
 
+    const isVoltageLevelEdit =
+        voltageLevelToEdit &&
+        voltageLevelToEdit.equipmentId === voltageLevelIdWatch;
+
```

Comment:
```suggestion
    const isVoltageLevelEdit =
        voltageLevelToEdit &&
        voltageLevelToEdit.equipmentId === voltageLevelIdWatch;

```

const isVoltageLevelEdit = voltageLevelToEdit?.equipmentId === voltageLevelIdWatch;